src/app/web-app.component.ts

```diff
@@ -113,6 +131,108 @@ export class WebAppComponent implements OnInit {
         verticalPosition: 'top'
       });
     });
-  }
 
+    // Keyboard Shortcuts Listener
+    this.unlisten = this.keyboardShortcuts.listen(
+      {
+        // Navigation Page
```

Comment:
Make similar change in every case.
```suggestion
        // Open Navigation Page
```